packages/bsp/common/usr/lib/armbian/armbian-audio-config

```diff
@@ -7,9 +7,9 @@
 [[ -z $(command -v aplay) ]] && echo "Missing aplay; doing nothing." && exit 0
 
 mixer_cmds() {
-	parm=${4:-on}
-	echo sset "$1" "$2" "$parm"
-	echo sset "$1" "$parm"
+        parm=${3:-on}
+        echo "sset '$1' '$2' '$parm'"
+        echo "sset '$1' '$parm'"
```

Comment:
Please keep the indent style.

Reply:
Ah, thanks for pointing that out. I had made this PR with the github webui because my build checkout was busy building another branch, but hadn't thought to check indentation.

Anyway, I fixed this with a force push.